visual-dotnet/SauceLabs.Visual/VisualCheckOptions.cs

```diff
@@ -13,5 +13,8 @@ public class VisualCheckOptions
         public IgnoreRegion[]? IgnoreRegions { get; set; }
         public IWebElement[]? IgnoreElements { get; set; }
         public bool? CaptureDom { get; set; }
+
+        public string? ClassName { get; set; }
```

Comment:
How about following the convention from webdriver?
```suggestion
        public string? SuiteName { get; set; }
```